src/sentry/search/events/datasets/spans_metrics.py

```diff
@@ -31,7 +30,6 @@ def search_filter_converter(
         self,
     ) -> Mapping[str, Callable[[SearchFilter], WhereType | None]]:
         return {
-            "message": self._message_filter_converter,
```

Comment:
was this intentionally removed?

Reply:
i'm not sure what's going on - some git bug